wouso/resources/static/css/cpanel-style.css

```diff
@@ -120,3 +120,20 @@ div.second {
 .form-control {
     width: 40%;
 }
+/* Groups */
+.right {
```

Comment:
@catalin-oancea I think there is a bootstrap class for that actually, called text-right